.travis.yml

```diff
@@ -70,7 +70,6 @@ deploy:
 
```

Comment:
There is still some warnings from travis:

Build config validation
- root: deprecated key sudo (The key `sudo` has no effect anymore.)
- deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
- deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
- root: missing os, using the default linux
- deploy: key upload-dir is not underscored, using upload_dir
- deploy: key local-dir is not underscored, using local_dir
- deploy: key upload-dir is not underscored, using upload_dir

Reply:
most of these warnings concerns the future version of deploy functionality ([dplv v2](https://docs.travis-ci.com/user/deployment-v2)) whiwh is for now in beta.
We currently use [dpl v1](https://docs.travis-ci.com/user/deployment), none of the following suggestions are documented:
- upload_dir  (upload-dir is)
- cleanup (skip_cleanup is)

I will push a change on local-dir which is not the key documented in v1, and on os/sudo key.